web/styles/popovers.css

```diff
@@ -833,14 +833,48 @@ ul {
         margin: 0;
     }
 
-    /* Override the default border-radius to properly align
-       the button corners with `stream_header_colorblock`. */
-    .dropdown-toggle {
-        border-radius: 1px 4px 4px 1px !important;
+    .move-topic-dropdown {
+        display: flex;
+        justify-content: flex-start;
+        margin-bottom: 10px;
+
+        .dropdown-widget-button {
+            /* Override the default border-radius to properly align
+            the button corners with `stream_header_colorblock`. */
+            border-radius: 1px 4px 4px 1px !important;
+            outline: none;
+            line-height: 24px;
+            width: auto;
+            max-width: 206px;
+        }
+
+        .stream_header_colorblock {
+            margin: 0;
+        }
+
+        #move_topic_stream_name {
+            overflow: hidden;
+            text-overflow: ellipsis;
+            white-space: nowrap;
+            color: var(--color-text-default);
+
+            .stream-privacy-type-icon {
+                font-size: 13px;
+                width: 13px;
+                height: 13px;
+                position: relative;
+                top: 2px;
+            }
+        }
+
+        .fa-chevron-down {
+            float: right;
+            padding-left: 5px;
+            color: hsl(0deg 0% 58%);
+            font-weight: lighter;
+        }
```

Comment:
There is some duplicate CSS here and in compose.css but I think it is better to let compose have its own copy of this since how we render this in compose will most like differ from how we render it in other places in future.

Reply:
I am planning to modify this CSS a bit, in next iteration, so we can do that then.